### PR TITLE
Juheehong dev

### PR DESCRIPTION
**Pull request by juheehong11**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java [hunk 1]
```diff
@@ -13,6 +13,6 @@ public class HelloWorldServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("Pikachu");
```
Comment (estefanytorres) [inline portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java]: Was this for testing purposes or do you want to keep it ?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -13,16 +13,42 @@
...
+    if(counter > 4){
+        counter = 1;
+    }
+}, 5000);
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+async function fetchPokemon() {
+    const serverResponse = await fetch('/hello');
```
Comment (estefanytorres) [inline portfolio/src/main/webapp/script.js:73]: I would suggest changing the servlet name "/hello" or creating a new one, I know this was probably for testing purposes but it's better not to leave code with names that don't correspond to the real function

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -13,16 +13,42 @@
...
+    counter++;
+    if(counter > 4){
+        counter = 1;
+    }
+}, 5000);
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+async function fetchPokemon() {
```
Comment (estefanytorres) [inline portfolio/src/main/webapp/script.js:72]: Maybe add a comment on top of the method to explain better what the function is meant for

**Review by estefanytorres — CHANGES_REQUESTED** [review]
Great job Juhee! I would only ask to clean up the get pokemon call, it is confusing that the servlet is named helloworld, feel free to remove this functionality, rename the servlet or create a new one with a descriptive name.

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java (hunk 1 above)
Comment (juheehong11) [reply portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java]: Oh! I guess I don't actually need it anymore, thank you for pointing that out!

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (juheehong11) [reply portfolio/src/main/webapp/script.js:73]: I changed it now, thank you for the feedback :)

**Pull request closed without merging**